tests/integration_tests.rs

```diff
@@ -84,51 +147,34 @@ async fn test_wascc_provider() -> Result<(), Box<dyn std::error::Error>> {
 
     assert_eq!(pod.status.unwrap().phase.unwrap(), "Pending");
 
-    let api = Api::namespaced(client, "default");
-    let inf: Informer<Pod> = Informer::new(api).params(
-        ListParams::default()
-            .fields("metadata.name=greet-wascc")
-            .timeout(30),
-    );
+    wait_for_pod_ready(client, "greet-wascc").await?;
 
-    let mut watcher = inf.poll().await?.boxed();
-    let mut went_ready = false;
-    while let Some(event) = watcher.try_next().await? {
-        match event {
-            WatchEvent::Modified(o) => {
-                let phase = o.status.unwrap().phase.unwrap();
-                if phase == "Running" {
-                    went_ready = true;
-                    break;
-                }
-            }
-            WatchEvent::Error(e) => {
-                panic!("WatchEvent error: {:?}", e);
-            }
-            _ => {}
-        }
-    }
+    Ok(())
+}
 
-    assert!(went_ready, "pod never went ready");
+struct WasccTestResourceCleaner {}
 
-    // Send a request to the pod to trigger some logging
-    reqwest::get("http://127.0.0.1:8080")
-        .await
-        .expect("unable to perform request to test pod");
+impl Drop for WasccTestResourceCleaner {
+    fn drop(&mut self) {
+        let t = std::thread::spawn(move || {
+            let mut rt =
+                tokio::runtime::Runtime::new().expect("Failed to reate Tokio runtime for cleanup");
```

Comment:
Nit: s/reate/create

Reply:
Ooh thanks - also fixed in the place I copied this from.  (Which was also me.)